docs/platforms/javascript/common/sourcemaps/uploading/angular-nx.mdx

```diff
@@ -6,29 +6,24 @@ supported:
   - javascript.angular
 ---
 
-If you're using [`@nx/angular`](https://nx.dev/packages/angular), you can use the Nx CLI together with our Sentry webpack plugin to automatically create releases and upload source maps to Sentry when building your app (with `nx build`, for example).
-Due to `@nx/angular`'s architecture, to register the webpack plugin, you'll first need to configure the [`@nx/angular:webpack-browser` executor](https://nx.dev/packages/angular/executors/webpack-browser).
+If you're using [`@nx/angular`](https://nx.dev/packages/angular), you can use the Nx CLI together with one of our bundler plugins (e.g. Webpack or esbuild) to automatically upload source maps to Sentry when building your app (with `nx build`, for example).
+Due to `@nx/angular`'s architecture, you'll first need to configure an executor that allows registering bundler plugins.
 In the end, you'll be able to automatically upload source maps whenever you're creating a production build of your app.
 
-## Install
-
-Install the Sentry webpack plugin:
-
-```shell {tabTitle:npm}
-npm install --save-dev @sentry/webpack-plugin
-```
+Configure your app to upload source maps in three steps.
 
-```shell {tabTitle:Yarn}
-yarn add --dev  @sentry/webpack-plugin
-```
+## 1. Configure `project.json`
 
-## Configure
+Before you register the Sentry bundler plugin, first ensure that you can register plugins (e.g. webpack or esbuild plugins) with your current executor (or Angular builder).
+Typically, this is possible if you have access to a `webpack.config.js` or a similar configuration file.
+If this already is the case, skip to [step 2](#2-register-the-sentry-webpack-plugin).
 
-Configure your app to upload source maps in three steps.
+If you cannot yet register a plugin, you'll need to change the executor in your `project.json` to a custom executor that allows registering a plugin. For example one of these:
 
-### 1. Configure `project.json`
+- [`@nx/angular:webpack-browser`](https://nx.dev/nx-api/angular/executors/webpack-browser)
+- [`@nx/angular:browser-esbuild`](https://nx.dev/nx-api/angular/executors/browser-esbuild)
 
-In your `project.json`, replace the default executor (`@angular-devkit/build-angular:browser`) with `@nx/angular:webpack-browser`:
+For example, in your `project.json`, replace the default executor (`@angular-devkit/build-angular:browser`) with `@nx/angular:webpack-browser`:
```

Comment:
> If you currently cannot register a plugin, you'll need to change the executor in your `project.json` to a custom executor (such as [`@nx/angular:webpack-browser`](https://nx.dev/nx-api/angular/executors/webpack-browser) or [`@nx/angular:browser-esbuild`](https://nx.dev/nx-api/angular/executors/browser-esbuild)) that allows registering a plugin. In your `project.json`, replace the default executor (`@angular-devkit/build-angular:browser`) with an appropriate custom executor. For example: 

Instead of breaking up the text with bullet points for the executors, I would combine lines 21-26 like above so that the flow of info is more connected